src/date.ts

```diff
@@ -87,9 +87,21 @@ const padZero = (n: number): string =>
 const time = (date: Date): string =>
     `${padZero(date.getUTCHours())}.${padZero(date.getUTCMinutes())}`;
 
+const localTime = (date: Date): string =>
+    `${padZero(date.getHours())}.${padZero(date.getMinutes())}`;
+
 const format = (date: Date): string =>
     `${day(date)} ${date.getUTCDate()} ${month(date)} ${date.getUTCFullYear()} ${time(date)} UTC`;
 
+const formatLocal = (date: Date): string =>
+    `${localDay(date)} ${date.getDate()} ${localMonth(date)} ${date.getFullYear()} ${localTime(date)}`;
```

Comment:
Nice! Dotcom also includes the timezone at the end of this (e.g. BST, EDT) - do we want to do that too?

Reply:
I think that would be a nice addition, I've pushed the following change. Couldn't see an easy way to display the shortened timezone
<img width="257" alt="Screen Shot 2020-11-02 at 12 34 09" src="https://user-images.githubusercontent.com/11618797/97868965-8b5cf100-1d08-11eb-9e15-e07277b81fc1.png">